packages/common/src/workflow-options.ts

```diff
@@ -38,24 +41,76 @@ export enum WorkflowIdReusePolicy {
   WORKFLOW_ID_REUSE_POLICY_REJECT_DUPLICATE = 3,
 
   /**
-   * Terminate the current workflow if one is already running.
```

Comment:
The deprecation comment needs the `@deprecated` annotation. Also should come last… though arguably, the fact that this is incompatible with `WorkflowIdConflictPolicy` should be part of the deprecation note.

Reply:
Suggestion:

```
  /**
   * Terminate the current Workflow if one is already running; otherwise allow reusing the Workflow ID.
   *
   * @deprecated Use {@link WORKFLOW_ID_REUSE_POLICY_ALLOW_DUPLICATE} instead, and
   *             set `WorkflowOptions.workflowIdConflictPolicy` to {@link WorkflowIdConflictPolicy.WORKFLOW_ID_CONFLICT_POLICY_TERMINATE_EXISTING}.
   *             When using this option, `WorkflowOptions.workflowIdConflictPolicy` must be left unspecified.
   */
```

Reply:
Much better thanks